package.json

```diff
@@ -18,6 +18,7 @@
   "license": "ISC",
   "dependencies": {
     "axios": "^0.21.1",
+    "babel-plugin-styled-components": "^1.12.0",
```

Comment:
Should be a `devDependency`. Use the `-D` (or `--save-dev`) option.

`npm rm babel-plugin-styled-components && npm i -D babel-plugin-styled-components`

Reply:
i changed